kubetest/main.go

```diff
@@ -212,6 +224,17 @@ func getDeployer(o *options) (deployer, error) {
 	switch o.deployment {
 	case "bash":
 		return newBash(&o.clusterIPRange), nil
+	case "dind":
+		// Once Kubernetes is built, we should grab the version for dind.
+		if o.deployment == "dind" && o.dindImage == "" {
```

Comment:
it's within `switch o.deployment`, only need to check `o.dindImage`

Reply:
That's a very good point.